endpoints.json

```diff
@@ -66,38 +66,68 @@
     "description": "adds a new comment to the specified article_id",
     "queries": [],
     "exampleRequest": {
-        "username": "butter_bridge",
-        "body": "What a wonderful day!"
+      "username": "butter_bridge",
+      "body": "What a wonderful day!"
     },
     "exampleResponse": {
-        "comment": {
-            "comment_id": 1,
-            "author": "butter_bridge",
-            "body": "What a wonderful day!",
-            "article_id": 1,
-            "votes": 0,
-            "created_at": "2024-11-26T12:00:00.000Z"
-        }
+      "comment": {
+        "comment_id": 1,
+        "author": "butter_bridge",
+        "body": "What a wonderful day!",
+        "article_id": 1,
+        "votes": 0,
+        "created_at": "2024-11-26T12:00:00.000Z"
+      }
     }
-},
-"PATCH /api/articles/:article_id": {
+  },
+  "PATCH /api/articles/:article_id": {
     "description": "updates the votes of an article by the specified amount",
     "queries": [],
     "exampleRequest": {
-        "inc_votes": 10
+      "inc_votes": 10
     },
     "exampleResponse": {
-        "article": {
-            "article_id": 1,
-            "title": "Seafood substitutions are increasing",
-            "topic": "cooking",
-            "author": "weegembump",
-            "body": "Text from the article...",
-            "created_at": "2018-05-30T15:59:13.341Z",
-            "votes": 110,
-            "article_img_url": "https://images.pexels.com/photos/97050/pexels-photo-97050.jpeg?w=700&h=700"
-        }
+      "article": {
+        "article_id": 1,
+        "title": "Seafood substitutions are increasing",
+        "topic": "cooking",
+        "author": "weegembump",
+        "body": "Text from the article...",
+        "created_at": "2018-05-30T15:59:13.341Z",
+        "votes": 110,
+        "article_img_url": "https://images.pexels.com/photos/97050/pexels-photo-97050.jpeg?w=700&h=700"
+      }
     }
-}
+  },
+  "DELETE /api/comments/:comment_id": {
+    "description": "deletes the specified comment by its comment_id",
+    "queries": [],
+    "exampleResponse": {
+      "status": 204
+    }
+  },
+  "GET /api/users": {
+    "description": "serves an array with all users",
+    "queries": [],
+    "exampleResponse": {
+        "users": [
+            {
+                "username": "butter_bridge",
+                "name": "jonny",
+                "avatar_url": "https://www.healthytherapies.com/wp-content/uploads/2016/06/Lime3.jpg"
+            },
+            {
+                "username": "icellusedkars",
+                "name": "sam",
+                "avatar_url":"https://avatars2.githubusercontent.com/u/24604688?s=460&v=4"
 
+            },
+            {
+                "username": "rogersop",
+                "name": "paul",
+                "avatar_url": "https://avatars2.githubusercontent.com/u/24604688?s=460&v=4"
+            }
+        ]
+    }
```

Comment:
This is the correct format we should be expecting. Body as an object with a key of users that holds the array of users